pkg/sinks/forward/forward.go

```diff
@@ -213,6 +213,14 @@ func (df *DataForward) forwardAChunk(ctx context.Context) {
 	// store the offsets of the messages we read from ISB
 	var readOffsets = make([]isb.Offset, len(readMessages))
 	for idx, m := range readMessages {
+		metrics.ReadBytesCount.With(map[string]string{
```

Comment:
It might be better to follow the pattern in `reduce/data_forward.go`, to use a variable to add them up, and then emit the metric outside the for loop.

Reply:
Could you please also update the other places (source, udf) to use the same patten?